### ISSUE TITLE
tpi - verifying checksum notification enhancement.

### ISSUE THREAD

**Issue by Xatinni** [turing-machines/tpi#25]
Is it possible to add a notification during the verifying checksum phase, eg. after an image has been written. example: in the attached screenshot it took over 4 minutes for checksum verification phase to complete, during this time users
 may think tpi has hung as there is no notification or visual indication on what's happening.

![image](https://github.com/turing-machines/tpi/assets/5786215/b553c7cf-73fe-4f74-a78f-967a14e68972)

**Comment by svenrademakers** [comment]
I agree the user cues are very minimal at the moment. The TPI tool should show something like 'verifying checksum ..'

**Comment by gjrtimmer** [comment]
@Xatinni can this be fixed in the `tpi` command? If so could you provide a PR?
